course-backend/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
remove this package json, it is not the one used for our project. The one used is under course-backend/course-content.
If you can try to recommit your project with only that folder, since it is the only one we need